### ISSUE TITLE
Contribute to Blackjax

### ISSUE THREAD

**Issue by junpenglao** [modichirag/GSM-VI#1]
Congratulations on the paper!

Would you be interested to contribute GSM-VI to Blackjax?

**Comment by modichirag** [comment]
Hey, thank you. 
Yes, I will be very interested in contributing this to Blackjax. I assume the way to do that would be via starting a PR on Blackjax github repo?

**Comment by junpenglao** [comment]
Yep! The VI algorithms are orgernized in https://github.com/blackjax-devs/blackjax/blob/main/blackjax/vi currently
